src/txn/txn_rollback_to_stable.c

```diff
@@ -1231,23 +1230,23 @@ static int
 __rollback_abort_updates(WT_SESSION_IMPL *session, WT_REF *ref, wt_timestamp_t rollback_timestamp)
 {
     WT_PAGE *page;
+    bool modified;
 
     /*
      * If we have a ref with clean page, find out whether the page has any modifications that are
      * newer than the given timestamp. As eviction writes the newest version to page, even a clean
      * page may also contain modifications that need rollback.
      */
     page = ref->page;
-    if (!__wt_page_is_modified(page) &&
-      !__rollback_page_needs_abort(session, ref, rollback_timestamp)) {
+    modified = __wt_page_is_modified(page);
+    if (!modified && !__rollback_page_needs_abort(session, ref, rollback_timestamp)) {
         __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session), "%p: page skipped", (void *)ref);
```

Comment:
Yes, `stable` is correct.

Reply:
Should we increment `txn_rts_tree_walk_skip_pages` here @kommiharibabu ?

Reply:
The tree walk logic already found this page because this page is in memory. Later we are verifying whether the page is modified or not to decide whether the page has unstable updates or not after taking the hazard pointer on the page. So I feel this stat may not be the correct one, I don't think we have any other stat to indicate the page skip.